### PR TITLE
Introduce define to change namespaces

### PR DESCRIPTION
**Pull request by ilyachur**
This PR is connected with these changes: https://github.com/openvinotoolkit/openvino/pull/6920

### CODE REVIEW

File: modules/arm_plugin/src/opset/utils.hpp [hunk 1]
```diff
@@ -26,7 +26,11 @@ arm_compute::ActivationLayerInfo makeActivationLayerInfo(ngraph::Node* node);
 }  // namespace opset
 }  // namespace ArmPlugin
 
+#ifndef OV_NEW_API
```
Comment (apankratovantonp) [inline modules/arm_plugin/src/opset/utils.hpp:29]: What about `OV_NGRAPH_NAMESPACE` macro :
```c++
#ifndef OV_NEW_API
#define OV_NGRAPH_NAMESPACE ngraph
#else
#define OV_NGRAPH_NAMESPACE ov
#endif

namespace OV_NGRAPH_NAMESPACE { 
...
} //  namespace OV_NGRAPH_NAMESPACE

```

Comment (ilyachur) [reply modules/arm_plugin/src/opset/utils.hpp]: Fixed

**Comment by ilyachur** [comment]
@apankratovantonp Is it ok that I have the next issue for ubuntu20:
```
CMake Error at /usr/local/share/cmake-3.13/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
 Could NOT find Java (missing: Java_JAR_EXECUTABLE Java_JAVAC_EXECUTABLE
Java_JAVADOC_EXECUTABLE) (found version "11.0.10")
```
???

**Comment by apankratovantonp** [comment]
> @apankratovantonp Is it ok that I have the next issue for ubuntu20:
> 
> ```
> CMake Error at /usr/local/share/cmake-3.13/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
>  Could NOT find Java (missing: Java_JAR_EXECUTABLE Java_JAVAC_EXECUTABLE
> Java_JAVADOC_EXECUTABLE) (found version "11.0.10")
> ```
> 
> ???
Yes, it is ok. This platform dose not even build ARM plugin.

File: modules/arm_plugin/src/opset/utils.hpp [hunk 2]
```diff
@@ -10,6 +10,12 @@
 #include "arm_compute/core/QuantizationInfo.h"
 #include "arm_compute/core/Types.h"
 
+#ifndef OV_NEW_API
```
Comment (apankratovantonp) [inline modules/arm_plugin/src/opset/utils.hpp:13]: Should this define be moved to OpenVINO? Or there is no such issues in other plugins and modules?

**Review by apankratovantonp — APPROVED** [review]

File: modules/arm_plugin/src/opset/utils.hpp (hunk 2 above)
Comment (ilya-lavrenov) [reply modules/arm_plugin/src/opset/utils.hpp:13]: @ilyachur looks like we don't have good solution for template likes Variant?

Comment (ilyachur) [reply modules/arm_plugin/src/opset/utils.hpp:13]: Yes it is right, we don't have a good backward compatible solution

Comment (ilyachur) [reply modules/arm_plugin/src/opset/utils.hpp:13]: Looks like different plugins and modules can use different ways to resolve the issue.

**Comment by ilyachur** [comment]
@apankratovantonp Could you have a look?

**Pull request merged**